example_custom_unpacking_test.go

```diff
@@ -0,0 +1,194 @@
+package tarantool_test
```

Comment:
![image](https://user-images.githubusercontent.com/20455996/161785835-4e2aedb9-6437-45e2-8549-7fba2b46358d.png)

Sentences are not separated by dot

Reply:
Fixed.